browser/net/brave_network_delegate_browsertest.cc

```diff
@@ -25,6 +27,22 @@
 
 using net::test_server::EmbeddedTestServer;
 
+bool NavigateRenderFrameToURL(content::RenderFrameHost* frame,
+                              std::string iframe_id,
+                              const GURL& url) {
+  std::string script = base::StringPrintf(
+      "setTimeout(\""
+      "var iframes = document.getElementById('%s');iframes.src='%s';"
+      "\",0)",
+      iframe_id.c_str(), url.spec().c_str());
```

Comment:
please confirm there's no way for any attacker controlled input to end up as part of this string thereby causing XSS

Reply:
@diracdeltas this is a test :)

Reply:
@bridiver yes i'm aware, just being paranoid. guessing the answer is no.

Reply:
there's nothing special about the script and it was actually copied from a chromium test helper and modified to use the render frame instead of the webcontents. It isn't related to the functionality here in any way, it's just a way to update the iframe url in the test